docs/proposals/local-cluster-ux.md

```diff
@@ -0,0 +1,184 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Kubernetes Local Cluster Experience
+
+This proposal attempts to improve the existing local cluster experience for kubernetes.
+The current local cluster experience is sub-par and often not functional.
+This proposal is a RFC!
+
+## Primary Goals
```

Comment:
Multinode is being addressed by the `kube-deploy` repo. This proposal is
meant to only focus on the local cluster experience.

On Mon, Apr 18, 2016 at 12:10 PM, Lucas Käldström <notifications@github.com>
wrote:

> In docs/proposals/local-cluster-ux.md
> <https://github.com/kubernetes/kubernetes/pull/24106#discussion_r60116660>
> :
>
> > +Documentation for other releases can be found at
> > +[releases.k8s.io](http://releases.k8s.io).
> > +</strong>
> > +--
> > +
> > +<!-- END STRIP_FOR_RELEASE -->
> > +
> > +<!-- END MUNGE: UNVERSIONED_WARNING -->
> > +
> > +# Kubernetes Local Cluster Experience
> > +
> > +This proposal attempts to improve the existing local cluster experience for kubernetes.
> > +The current local cluster experience is sub-par and often not functional.
> > +This proposal is a RFC!
> > +
> > +## Primary Goals
>
> Although I know you focus on local cluster only, should we mention
> networking at least (long term goals or something)?
> Otherwise the user probably wonders what will happen to docker-multinode
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24106/files/4d4288e76854d11c1f2db2ffd002a3ed041995fe#r60116660>
>